mobile/src/main/java/rs/readahead/washington/mobile/views/activity/onboarding/OnBoardingActivity.kt

```diff
@@ -74,10 +74,13 @@ class OnBoardingActivity : BaseActivity(), OnBoardActivityInterface,
         if (isOnboardLockSet) {
             Preferences.setFirstStart(false)
```

Comment:
inside OnBordingActivity -> consider cleaning the code, `backBtn and nextBtn` is not used so remove it.

this ` var fragment: Fragment = when (position) {
                ONBOARDING_INTRODUCTION_VIEW_INDEX -> OnBoardIntroFragment()
                ONBOARDING_CAMERA_VIEW_INDEX -> OnBoardCameraFragment()
                ONBOARDING_RECORDER_VIEW_INDEX -> OnBoardRecorderFragment()
                ONBOARDING_FILES_VIEW_INDEX -> OnBoardFilesFragment()
                ONBOARDING_LOCK_VIEW_INDEX -> OnBoardLockFragment()
                else -> OnBoardIntroFragment()
            }`. this should be a val not a var is it's an immutable.